app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt

```diff
@@ -78,6 +82,10 @@ class FaceTrackingViewModel : ViewModel(), KoinComponent {
                         liveAdjustedVector.postValue(oldVector)
                     }
                     else -> {
+                        val isTablet = get<Context>().resources.getBoolean(R.bool.is_tablet)
```

Comment:
We should grab this value in the `init` block. Reading from the resources files can be expensive, and we don't want to do it every time we loop through here.